.github/workflows/ruby.yml

```diff
@@ -14,6 +14,7 @@ on:
     branches: ['master']
   schedule:
     - cron: '0 0 * * 0'
```

Comment:
please consider carefully if you might need to remove this cron.. If you maintain this, the testing will be run once a week and last time, the test failed unexpectedly while nothing has changed.

You might want to investigate those as well

Reply:
Added `workflow_dispatch` trigger to run actions manually to resolve intermittent failures.